### PR TITLE
Fix DoctrineDBAL210Set

### PR DESCRIPTION
**Pull request by kick-the-bucket**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
There you go :+1:  :wink: 

I'll check after the CI is green. Let me know if you need re-trigger.

**Comment by kick-the-bucket** [comment]
Code style should be fixed

**Review by samsonasik — APPROVED** [review]

File: tests/bootstrap.php [hunk 1]
```diff
@@ -6,9 +6,10 @@
 
 // autoload rector first, but with local paths
 // build preload file to autoload local php-parser instead of phpstan one, e.g. in case of early upgrade
-exec(PHP_BINARY.' vendor/rector/rector-src/build/build-preload.php '.__DIR__.'/../');
+$preloadPath = __DIR__ . '/../';
+exec(PHP_BINARY . ' vendor/rector/rector-src/build/build-preload.php ' . $preloadPath);
 sleep(3);
 
-require __DIR__ . '/../preload.php';
+require $preloadPath . 'preload.php';
```
Comment (TomasVotruba) [inline tests/bootstrap.php:13]: This hides the absolute path and breaks PHPStorm autocomplete. 
In this case, just add it to `phpstan.neon`, as this path is generated dynamically to test new php-parser 5.

**Comment by TomasVotruba** [comment]
Thank you :clap:

**Pull request merged**